powersimdata/input/change_table.py

```diff
@@ -124,6 +125,10 @@ class ChangeTable:
         keys in the dictionary are: *'lat'*, *'lon'*, one of *'zone_id'*/*'zone_name'*,
         and optionally *'Pd'*, specifying the location of the bus, the demand zone, and
         optionally the nominal demand at that bus (defaults to 0).
+    * *'remove_branch'*:
```

Comment:
do we want `remove_branch` (`remove_bus`) following `new_branch` (`new_bus`) or all `new_x` and `remove_x` altogether? I have no preference.